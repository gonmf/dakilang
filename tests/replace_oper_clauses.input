
add(X, Y, Z).
add(X, Y, Z)~

