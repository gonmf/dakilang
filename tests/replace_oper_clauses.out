>
> add(X, Y, Z).
Built-in operator clause already exists

> add(X, Y, Z)~
Built-in operator clause cannot be removed

>
